server/game/cards/events/06/plunder.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Plunder extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Gain gold',
+            condition: () => this.getGold() >= 1,
+            cost: ability.costs.kneelFactionCard(),
+            handler: () => {
+                let gold = this.getGold();
+                this.game.addGold(this.controller, gold);
+                this.game.addMessage('{0} uses {1} and kneels their faction card to gain {2} gold', this.controller, this, gold);
+            }
+        });
+    }
+
+    getGold() {
+        let opponent = this.game.getOtherPlayer(this.controller);
+
+        if(!opponent) {
+            return;
```

Comment:
You're correct, I missed the condition. Still glad you changed it though, it was kind of odd regardless.